eng/script-tests/Repo-File-Content-Replacements/Repo-File-Content-Replacements.Tests.ps1

```diff
@@ -0,0 +1,60 @@
+<#
+```
+Invoke-Pester -Output Detailed $PSScriptRoot/Repo-File-Content-Replacements.Tests.ps1
+```
+#>
+
+Import-Module Pester
+Set-StrictMode -Version Latest
+
+BeforeAll {
+    function Backup-Folder($targetFolder, $backupFolder) {
+        if (!(Test-Path $targetFolder)) {
+            return $null
+        }
+        
+        if (!(Test-Path $backupFolder)) {
+            New-Item $backupFolder -ItemType Directory
+        }
+        Copy-Item -Path "$targetFolder/*" -Destination $backupFolder -Recurse -PassThru -Force
+    }
+    function Reset-Folder($backupFolder, $targetFolder) {
```

Comment:
Do the functions have to be declared within a BeforeAll to be in scope or something?

Reply:
It is a helper function. Let me try to move it out to script body.

Reply:
Tested. This is required. Test helper function should put in "Before All".